lib/utils.js

```diff
@@ -1,12 +1,23 @@
 'use strict';
 
 const EventEmitter = require('events').EventEmitter;
-const exec = require('child_process').exec;
 const qs = require('qs');
 const crypto = require('crypto');
 
 const hasOwn = (obj, prop) => Object.prototype.hasOwnProperty.call(obj, prop);
 
+// Certain sandboxed environments (our known example right now are CloudFlare
+// Workers) may make `child_process` unavailable. Because `exec` isn't critical
+// to the operation of stripe-node, we handle this unavailability gracefully.
+var exec = null;
```

Comment:
minor style nit
```suggestion
let exec = null;
```

Reply:
Oops, even I should know about that one. Thanks!